main.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative './Models/author'
+require_relative './Models/book'
+require_relative './Models/order'
+require_relative './Models/reader'
+require_relative './Models/library'
+require_relative 'seeds'
+
+library = Library.new([], [], [], [])
+file_name = 'settings.yml'
+
+Seeds.fill_the_library(library, file_name)
+
+author4 = Author.new('Chuck Palahniuk', 'bio')
+library.authors << author4
+book10 = Book.new('Fight club', author4)
+library.books << book10
+reader4 = Reader.new({ name: 'DT', email: 'ds@gmail.com', city: 'D', street: 'street', house: 1 })
+library.readers << reader4
+order9 = Order.new(book10, reader4)
+library.orders << order9
+
+library.write_to_yaml(file_name)
+
+library1 = Library.new([], [], [], [])
```

Comment:
You already have `library` why do you need another instance? Even if you do choose better name for it

Reply:
fixed